/**:
  ros__parameters:
    camera_names: [ "camera_1","camera_2","camera_3" ]
    camera_1:
      frame_id: lucid_vision
      pixel_format: bgr8
      serial_no: 210900910
      fps: 15
      width: 1440
      height: 930
      resize_image: true
      camera_info_url: "/home/volt/projects/volt_drivers_ws/src/arena_camera/config/camera0.yaml"
    camera_2:
      frame_id: lucid_vision
      pixel_format: bgr8
      serial_no: 210900347
      fps: 10
      width: 1440
      height: 930
      resize_image: true
      camera_info_url: "/home/volt/projects/volt_drivers_ws/src/arena_camera/config/camera1.yaml"
    camera_3:
      frame_id: lucid_vision
      pixel_format: bgr8
      serial_no: 210800157
      fps: 10
      width: 1440
      height: 930
      resize_image: false
      camera_info_url: "/home/volt/projects/volt_drivers_ws/src/arena_camera/config/camera2.yaml"

